tests/testthat/test.CNode.R

```diff
@@ -8,7 +8,7 @@ test_that("CNode constructors", {
   expect_match(cnProd@endpoint, "https://cn.dataone.org/cn")
   expect_match(cnProd@endpoint, "https://cn.dataone.org/cn")
   # Skip unstable test environments.
-  expect_match(cnStaging2@endpoint, "https://cn-stage-2.test.dataone.org/cn")
+  expect_match(cnStaging2@endpoint, "https://cn-stage.test.dataone.org/cn")
```

Comment:
Is this right? Seems like `cnStaging2@endpoint` should be renamed to avoid confusion if you are switching to using the cn-stage environment.